### ISSUE TITLE
W20150830-14:02:32.215(8) (blaze.js:71) Warning: Blaze.render without a parent element is deprecated. You must specify where to insert the rendered content.

### ISSUE THREAD

**Issue by Ajaxsoap** [loredanacirstea/meteor-tabular-filter#2]
Hi Loredana,

Thanks for this package! The community needs this, as there are few packages out there, some are not maintained any more.

Based on the issue title, I think the main culprit after deleting the filter, for the filter to appear again is to reload, is this warning, [Blaze.render](http://docs.meteor.com/#/full/blaze_render) **_without parent is deprecated**_. I think by fixing this, the refresh action will be eliminated, I am not sure how to do this but it's worth trying.

**Comment by loredanacirstea** [comment]
Fixed this. Hope the package helps, even though I don't have time for maintaining it properly.

**Comment by Ajaxsoap** [comment]
Thanks Loredana for the quick fix! :+1: 
It works now, no need to reload page after removing the filters (hit X button)! 

However, I've noticed, for example, I've filtered a title "Fly", the filter works, then cleared the input and hit filter button, nothing happens, the last filter is always displayed on the table even if no value to be filtered with, in this case the title "Fly". But if i removed the filter (hit the X button), the table is back to it's normal listing or last filtered state.
